rpc/connection.go

```diff
@@ -608,6 +623,9 @@ type connectionClient struct {
 var _ GenericClient = connectionClient{}
 
 func (c connectionClient) Call(ctx context.Context, s string, args interface{}, res interface{}) error {
+	if isWithFireNow(ctx) || c.conn.initialReconnectBackoffWindow != 0 {
```

Comment:
Ah you are right; it's supposed to be `&&`. Now I'm confused why my tests worked ... :'(